modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/util/RexUtils.java

```diff
@@ -682,6 +663,10 @@ public static List<SearchBounds> buildHashSearchBounds(
             return null; // No bounds.
         }
 
+        if (!allowRange) {
```

Comment:
minor: seems it can be moved a bit upper - right before comment: "// Range bounds."

Reply:
added similar if-return to mentioned placed